### ISSUE TITLE
Some dates differ from date in post title

### ISSUE THREAD

**Issue by mgcth** [Ingenjorsarbete-For-Klimatet/ingenjorsarbete-for-klimatet.github.io#89]
## Description

See: https://github.com/Ingenjorsarbete-For-Klimatet/ingenjorsarbete-for-klimatet.github.io/blob/main/content/posts/ifk-smhi-2023-03-29.md?plain=1

**Comment by docNord** [comment]
Yes - I think this is because I have frequently started writing articles at one point but failed to finalize the work the same day, and therefore sometimes (when I have remembered) changed the date to more accurately match the actual publication date, based on the assumption that the publication date is more accurate as a statistic than the date at which work was initiated on the article.

What do you propose concerning this as a way of working going forwards @mgcth ?

**Comment by docNord** [comment]
Should we re-name files such that there is no date in them? Lately, I have tried this to some extent. Since the same project may well warrant several articles though, it is convenient to date them to make it easy to sort them in chronological order just from the file names.
